rust/src/lib.rs

```diff
@@ -237,7 +255,7 @@ impl<'a> BaseFunction<'a> for ImageToImage<'a> {
         &mut self.common
     }
     fn generate(&self) -> Result<(), WasmedgeSdErrno> {
-        if self.common.prompt.is_empty() {
+        if self.common.prompt.is_empty(){
```

Comment:
We should have a space here. Would you like to run cargo-fmt to fix the linter issue?

Reply:
I have fixed it; please check. Thx.